pypixplore/local.py

```diff
@@ -16,3 +17,17 @@ def show(self, name=None):
 
     def upgradeable(self):
         raise NotImplementedError
+
+    def package_status(self, package_name):
+        """
+        Check whether package_name is installed. If so, returns its version
+        :param package_name: str to be consulted by function
+        :return:  if installed - returns tuple with name of package and version
+                  if not installed - returns 0
+        """
+        pack_list = self.installed
+        for item in pack_list:
+            name_version = str(item).split(' ')
+            if package_name == name_version[0]:
+                return tuple(name_version)
+        return 0
```

Comment:
No need to return 0. Functions which end without hitting a return statement return None